src/locale/locale-en.json

```diff
@@ -477,34 +479,39 @@
     "MY_PROFILE": {
         "MY_PROFILE": "My Profile",
         "CHANGE_PASSWORD": "Change Password",
+        "SELECTED_TEST_LEVEL": "Selected Test Level",
+        "MATH_LEVEL_1": "Math Level 1",
```

Comment:
it is hard for me to say it should be in user profile ....this is very app spesific